History of research (#648)
faktaoklimatu/web: _infografiky/teploty/historie-sklenikoveho-efektu.md

---
layout:     infographic
title:      "Historie výzkumu skleníkového efektu"
slug:       "historie-sklenikoveho-efektu"
redirect_from: "/historie-sklenikoveho-efektu"
published:  2020-09-20
weight:     110
tags-scopes: [ svet ]
tags-topics: [ klima ]
caption:    " Skleníkový efekt je zkoumán více než 100let. První výpočty síly skleníkového efektu se objevily v devatenáctém století, měření a výpočty v průběhu dvacátého století je dále zpřesnily."
dataset:    
---

## Přehledové publikace

Tato časová osa vychází z těchto dvou přehledových publikací o výzkumu klimatu a klimatických změn: 
 * S.R. Weart: [The Discovery of Global Warming](https://www.hup.harvard.edu/catalog.php?isbn=9780674031890) [Online verze](https://history.aip.org/climate/method.htm) 

 * D.Archer, R.Pierrehumbert: [The Warming Papers](https://www.wiley.com/en-us/The+Warming+Papers%3A+The+Scientific+Foundation+for+the+Climate+Change+Forecast-p-9781405196161). 

## Více o jednotlivých událostech a objevech

> * **1824:** *[J.B Fouriere](https://en.wikipedia.org/wiki/Joseph_Fourier#Discovery_of_the_greenhouse_effect)* ze známé vzdálenosti Země od slunce a měření intenzity slunečního záření vypočítal, jaká by měla být průměrná teplota Země, kdyby byla pouze ohřívána sluncem. Protože mu vycházela teplota nižší, než jaká je skutečně měřená, uvažoval o dalších možných zdrojích tepla (např. o kosmickém záření), nebo o možném izolačním efektu atmosféry. Jakým mechanismem atmosféra může zachycovat teplo objevil o 35 let později John Tyndall. 

Reviewer: ```suggestion
* **1824:** *[J.B Fouriere](https://en.wikipedia.org/wiki/Joseph_Fourier#Discovery_of_the_greenhouse_effect)* ze známé vzdálenosti Země od Slunce a měření intenzity slunečního záření vypočítal, jaká by měla být průměrná teplota Země, kdyby byla pouze ohřívána Sluncem. Protože mu vycházela teplota nižší, než jaká je skutečně měřená, uvažoval o dalších možných zdrojích tepla (např. o kosmickém záření), nebo o možném izolačním efektu atmosféry. Jakým mechanismem atmosféra může zachycovat teplo objevil o 35 let později John Tyndall. 
```

* **1859:** *[John Tyndall](https://en.wikipedia.org/wiki/John_Tyndall#Molecular_physics_of_radiant_heat)* zkoumá pohlcování tepelného záření různými plyny. Jako první správně změřil absorpční koeficienty dusíku, kyslíku, oxidu uhličitého a dalších plynů a dochází k formulaci principů skleníkového efektu.

> * **1800 - 1880:** [Průmyslová revoluce](https://en.wikipedia.org/wiki/Industrial_Revolution) . Ačkoliv byl [parní stroj](https://en.wikipedia.org/wiki/Steam_engine) vynalezen roku 1698, trvalo ještě téměř celé století než se stal široce využívaným. Svou první aplikaci získal jako pumpa k odčerpávání vody z uhelných dolů a teprve později začal být používán k dopravě (vlaky a parní lodě) a k pohonu strojů v tehdy rychle rostoucím textilním průmyslu. Průmyslová revoluce probíhala v různých částech světa v různé době - začala v Anglii  a postupně se šířila v kontinentální Evropě a dále do USA, Japonska a kolonií. Období od roku 1850, kdy došlo k velkému rozšiřování výroby oceli, se někdy označuje jako druhá průmyslová revoluce. **Rozvoj těžby uhlí, parní stroje, železnice způsobují nárůst emisí oxidu uhličitého a zvyšování jeho koncentrací v atmosféře.** 

Reviewer: ```suggestion
* **1800 - 1880:** [Průmyslová revoluce](https://en.wikipedia.org/wiki/Industrial_Revolution) - ačkoliv byl [parní stroj](https://en.wikipedia.org/wiki/Steam_engine) vynalezen roku 1698, trvalo ještě téměř celé století než se stal široce využívaným. Svou první aplikaci získal jako pumpa k odčerpávání vody z uhelných dolů a teprve později začal být používán k dopravě (vlaky a parní lodě) a k pohonu strojů v tehdy rychle rostoucím textilním průmyslu. Průmyslová revoluce probíhala v různých částech světa v různé době - začala v Anglii a postupně se šířila v kontinentální Evropě a dále do USA, Japonska a kolonií. Období od roku 1850, kdy došlo k velkému rozšiřování výroby oceli, se někdy označuje jako druhá průmyslová revoluce. **Rozvoj těžby uhlí, parní stroje, železnice způsobují nárůst emisí oxidu uhličitého a zvyšování jeho koncentrací v atmosféře.** 
```

> * **1896:** *[Svante Arrhenius](https://en.wikipedia.org/wiki/Svante_Arrhenius#Greenhouse_effect)* publikuje [první výpočet](https://www.tandfonline.com/doi/abs/10.1080/14786449608620846) síly skleníkového efektu. Pomocí měření absorpce infračerveného záření měsíce zjišťuje, jak velká část je pohlcována vodní parou a oxidem uhličitým v atmosféře a odvozuje, že každé zdvojnásobení koncentrací oxidu uhličitého by mělo vést ke zvýšení teploty o stejnou teplotu. (neboli že teplota závisí na logaritmu koncentrace).Jeho výpočet je z dnešního pohledu spíše odhad, protože nijak nemodeluje proudění vzduchu a vody v atmosféře a řeší pouze změnu teploty vlivem změny koncentrací CO<sub>2</sub> a vodní páry. Citlivost klimatu - tedy hodnota, o kterou se zvýší průměrná teplota planety při zdvojnásobení koncentrací v CO<sub>2</sub> - mu vychází 4 °C. 

Reviewer: ```suggestion
* **1896:** *[Svante Arrhenius](https://en.wikipedia.org/wiki/Svante_Arrhenius#Greenhouse_effect)* publikuje [první výpočet](https://www.tandfonline.com/doi/abs/10.1080/14786449608620846) síly skleníkového efektu. Pomocí měření absorpce infračerveného záření měsíce zjišťuje, jak velká část je pohlcována vodní parou a oxidem uhličitým v atmosféře a odvozuje, že každé zdvojnásobení koncentrací oxidu uhličitého by mělo vést ke zvýšení teploty o stejnou teplotu (neboli že teplota závisí na logaritmu koncentrace). Jeho výpočet je z dnešního pohledu spíše odhad, protože nijak nemodeluje proudění vzduchu a vody v atmosféře a řeší pouze změnu teploty vlivem změny koncentrací CO<sub>2</sub> a vodní páry. Citlivost klimatu - tedy hodnota, o kterou se zvýší průměrná teplota planety při zdvojnásobení koncentrací v CO<sub>2</sub> - mu vychází 4 °C. 
```

* **1960:** *[Charles Keeling](https://en.wikipedia.org/wiki/Charles_David_Keeling#Work_with_Scripps_Institution_of_Oceanography,_1958%E2%80%932005)* publikuje [první výsledky](https://scrippsco2.ucsd.edu/assets/publications/keeling_tellus_1960.pdf) svých měření koncentrací CO<sub>2</sub> v atmosféře. Ačkoliv se dlouho vědělo o tom, že se koncentrace oxidu uhličitého v atmosféře zvyšují, nebyla známa metoda, jak je změřit přesně. S tou přišel až Keeling v roce 1956 a v roce 1958 začíná budovat měřicí stanici na Mauna Loa na Hawai, uprostřed Tichého oceánu. Data z měření, která pokračují dodnes, jsou dostupná na stránkách [Scrippsova Oceanografického Institutu](https://scrippsco2.ucsd.edu/data/atmospheric_co2/primary_mlo_co2_record.html) a příběh Keelinových měření v [článku (Anal.Chem. 2010)](https://pubs.acs.org/doi/pdf/10.1021/ac1001492). Keelingův článek [Atmospheric carbon dioxide variations at Mauna Loa Observatory](https://onlinelibrary.wiley.com/doi/abs/10.1111/j.2153-3490.1976.tb00701.x) z roku 1976, shrnující výsledky šestnácti let měření koncentrací CO<sub>2</sub>, se stal jedním z nejvíce citovaných článků v klimatologii. 

> * **1967:** *Syokuro Manabe* a *Richard T. Wetherald* publikují výsledky počítačového modelování klimatu a odhadu citlivosti klimatu. Jejich článek [Thermal Equilibrium of the Atmosphere with a Given Distribution of Relative Humidity](https://journals.ametsoc.org/jas/article/24/3/241/17328/Thermal-Equilibrium-of-the-Atmosphere-with-a-Given) se stal nejcitovanějším článkem v klimatologii. Slo o první práci, kdy byl počítačový model zemského klimatu použit k odhadu důsledků zdvojnásobení CO<sub>2</sub> na světovou teplotu. Práce použila výrazně realističtější model zpětné vazby vodní páry (_water vapour feedack_) než kterékoliv předchozí výzkumy. 

Reviewer: moc nevím, proč je tam anglický překlad v závorce - dával by mi smysl, kdyby to byl třeba odkaz o tom fenoménu, jinak nevím, proč tam je

> * **1969:** Přistání člověka na měsíci. Vesmírný program přinesl nejen výzkum prostoru mimo Zemi, od nízkých oběžných drah až po měsíc, ale také jiný pohled na naši planetu. Skrze fotky astronautů lidé poprvé vidí planetu jako celek, zjišťují, že při pohledu z vesmíru vypadá jako "modrá planeta". Ikonickou fotkou z té doby je [východ Země](https://en.wikipedia.org/wiki/Earthrise) nad povrchem měsíce. 

Reviewer: ```suggestion
* **1969:** Přistání člověka na Měsíci. Vesmírný program přinesl nejen výzkum prostoru mimo Zemi, od nízkých oběžných drah až po Měsíc, ale také jiný pohled na naši planetu. Skrze fotky astronautů lidé poprvé vidí planetu jako celek, zjišťují, že při pohledu z vesmíru vypadá jako "modrá planeta". Ikonickou fotkou z té doby je [východ Země](https://en.wikipedia.org/wiki/Earthrise) nad povrchem měsíce. 
```

* **1969:** Satelity umožňují přesné měření teploty atmosféry ve velkých výškách a nad oceány. [První meteorologické družice](https://en.wikipedia.org/wiki/Weather_satellite#History) byly vypouštěny na oběžnou dráhu v průběhu šedesátých let a postupně se podařilo měřeními teplot pokrýt oceány i vyšší vrstvy troposféry. Tyto data umožňily přesnější předpovědi počasí např. pro lodní dopravu, ale také přesnější stanovení rychlosti globálního oteplování. 

* **1972:** [Vrty z ledovcových jader](https://en.wikipedia.org/wiki/Ice_core#History) v Antarktidě a Grónsku ukazují historické koncentrace CO<sub>2</sub>, metanu a dallších plynů a dokládají jejich roli ve střídání dob ledových a meziledových. Zobrazení dat o koncentracích CO<sub>2</sub> za posledních 800 000 let najdete v [naší grafice](https://faktaoklimatu.cz/infografiky/koncentrace-co2). 

> * **1975:** Objev ozónové díry. Okolo roku 1975 několik [výzkumých týmů](https://en.wikipedia.org/wiki/Ozone_depletion#Research_history)  konstatovalo, že chlor-fluorované uhlovodíky (CFC, freony) se kumulují a mění reakce ve stratosféře, čímž dochází k úbytku ozónu. Za tyto objevy byla v roce 1995 udělena Nobelova cena za Chemii.  

Reviewer: ```suggestion
* **1975:** Objev ozónové díry. Okolo roku 1975 několik [výzkumých týmů](https://en.wikipedia.org/wiki/Ozone_depletion#Research_history)  konstatovalo, že chlor-fluorované uhlovodíky (CFC, freony) se kumulují a mění reakce ve stratosféře, čímž dochází k úbytku ozónu. Za tyto objevy byla v roce 1995 udělena Nobelova cena za chemii.  
```

* **1979:** Akademie věd USA zveřejňuje tzv. [Charneyho zprávu](https://faktaoklimatu.cz/studie/1979_charneyho-zprava), která na základě počítačových modelů dochází k závěru, že zdvojnásobení koncentrací CO<sub>2</sub> povede k oteplení o 1,5-4,5 °C se střední hodnotou 3 °C. V podstatě jde o potvrzení Arrheniova odhadu citlivosti klimatu z roku 1896 při započítání dynamických procesů v atmosféře a oceánech, které počítačové modely dokážou modelovat. 

> * **1981:** Rozvoj [osobních počítačů](https://en.wikipedia.org/wiki/History_of_personal_computers) probíhal v osmdesátých letech - v roce 1981 uvedla firma IBM na trh svoje první PC. Spolu s pozdějším rozvojem internety a dalších technologií významně napomohl tomu, že ekonomický růst přestává být výlučně závislý na energetice a spotřebě surovin a začíná docházet k tzv. [decouplingu](https://www.enviwiki.cz/wiki/Decoupling)

Reviewer: ```suggestion
* **1981:** Rozvoj [osobních počítačů](https://en.wikipedia.org/wiki/History_of_personal_computers) probíhal v osmdesátých letech - v roce 1981 uvedla firma IBM na trh svoje první PC. Spolu s pozdějším rozvojem internetu a dalších technologií významně napomohl tomu, že ekonomický růst přestává být výlučně závislý na energetice a spotřebě surovin a začíná docházet k tzv. oddělení ("[decoupling](https://www.enviwiki.cz/wiki/Decoupling)")
```

* **1982:** Společnost Exxon Mobile vydává interní zprávu [CO<sub>2</sub> Greenhouse effect - technical review](https://insideclimatenews.org/sites/default/files/documents/1982%20Exxon%20Primer%20on%20CO2%20Greenhouse%20Effect.pdf). V ní mimo jiné konstatuje, že zdvojnásobení koncentrací CO<sub>2</sub> povede k oteplení o 1,5 - 4 °C a odhaduje, že světové teploty budou v roce 2020 průměrně o 1 °C vyšší, než v roce 1960. Jde o jeden z důkazů, že informace o důsledcích spalování fosilních paliv byly těžařským společnostem dlouho známé.   

* **1985:** Mezinárodní vědecká [konference ve Villachu](http://what-when-how.com/global-warming/villach-conference-global-warming/) deklaruje shodu mezi experty na tom, že globální oteplování se děje a měly by být přijaty mezinárodní dohody omezující emise CO<sub>2</sub>.

> * **1987:** [Montrealský protokol](https://en.wikipedia.org/wiki/Montreal_Protocol), mezinárodní dohoda podepsaná roku 1987, omezuje emise látek narušujících ozonovou vrstvu. Dohoda byla úspěšná a vedla postupně k zastavení produkce většiny látek, které poškozují ozonovou vrstvu. Jak ukazují současná měření https://ozonewatch.gsfc.nasa.gov/ ozonová díra se v roce 2019 začala zmenšovat (tedy asi 40 let po objevu ozonové díry a asi 30 let po přijetí dohody). Montrealský protokol je často udávan jako příklad úspěšné mezinárodní snahy o ochranu klimatu. 

Reviewer: ```suggestion
* **1987:** [Montrealský protokol](https://en.wikipedia.org/wiki/Montreal_Protocol), mezinárodní dohoda podepsaná roku 1987, omezuje emise látek narušujících ozonovou vrstvu. Dohoda byla úspěšná a vedla postupně k zastavení produkce většiny látek, které poškozují ozonovou vrstvu. Jak ukazují [současná měření](https://ozonewatch.gsfc.nasa.gov/) ozonová díra se v roce 2019 začala zmenšovat (tedy asi 40 let po objevu ozonové díry a asi 30 let po přijetí dohody). Montrealský protokol je často udáván jako příklad úspěšné mezinárodní snahy o ochranu klimatu. 
```

* **1988:** Ralph Keeling objevil [metodu přesného měření koncentrace kyslíku v atmosféře](http://bluemoon.ucsd.edu/publications/ralph/34_PhDthesis.pdf) a jeho měření ukazují, že spalování fosilních paliv nejen zvyšuje koncentrace CO<sub>2</sub>, ale také snižuje atmosférické koncentrace kyslíku. Souvislosti ukazujeme v naší grafice [Cykly koncentrací CO₂ a O₂ v atmosféře](https://faktaoklimatu.cz/infografiky/cykly-koncentrace-co2).

> * **2018:** [Speciální zpráva IPCC SR15](https://www.ipcc.ch/sr15/) podrobně shrnuje výsledky výzkumů o dosažitelnosti hranice oteplení o 1.5 °C a srovnává dopady oteplení o 1,5 °C s očekávanými doapdy  otepelní o 2°C.  Dochází k závěru, že při rychlém snížení emisí skleníkových plynů je stále ještě možné udržet oteplení pod 1,5 °C.

Reviewer: ```suggestion
* **2018:** [Speciální zpráva IPCC SR15](https://www.ipcc.ch/sr15/) podrobně shrnuje výsledky výzkumů o dosažitelnosti hranice oteplení o 1,5 °C a srovnává dopady oteplení o 1,5 °C s očekávanými dopady  oteplení o 2 °C.  Dochází k závěru, že při rychlém snížení emisí skleníkových plynů je stále ještě možné udržet oteplení pod 1,5 °C.
```

> * **2020:** [Nejnovější výzkum](https://agupubs.onlinelibrary.wiley.com/doi/10.1029/2019RG000678) podrobně analyzuje změny klimatu v historii a zpřesňuje očekávané rozmezí <glossary id="citlivost">citlivosti klimatu</glossary> na 2,6 - 4,1°C.  

Reviewer: ```suggestion
* **2020:** [Nejnovější výzkum](https://agupubs.onlinelibrary.wiley.com/doi/10.1029/2019RG000678) podrobně analyzuje změny klimatu v historii a zpřesňuje očekávané rozmezí <glossary id="citlivost">citlivosti klimatu</glossary> na 2,6 - 4,1 °C.  
```

## Další odkazy a poznámky

* [Analýza citovanosti](https://www.carbonbrief.org/the-most-influential-climate-change-papers-of-all-time) z časopisu Carbon Brief, ukazuje, že nevlivnější, tedy nejvíce citované články o klimatu jsou 
    * Manabe, Wetherald: [Thermal Equilibrium of the Atmosphere with a Given Distribution of Relative Humidity](https://journals.ametsoc.org/jas/article/24/3/241/17328/Thermal-Equilibrium-of-the-Atmosphere-with-a-Given) (1967). Jde o první práci, kdy byl počítačový model zemského klimatu použit k odhadu důsledků zdvojnásobení CO<sub>2</sub> na světovou teplotu. Práce použila výrazně realističtější model zpětné vazby vodní páry (_water vapour feedack_) než kterékoliv předchozí výzkumy. 

>     * Keeling: [Atmospheric carbon dioxide variations at Mauna Loa Observatory](https://onlinelibrary.wiley.com/doi/abs/10.1111/j.2153-3490.1976.tb00701.x)(1976). článek ukazuje výsledky šestnácti let měření koncentrací CO<sub>2</sub> na Mauna Loa, ukazuje dlouhodobý nárůst koncentrací a je tedy první zprávou o velikosti efektu spalování fosilních paliv na složení atmosféry.   

Reviewer: ```suggestion
    * Keeling: [Atmospheric carbon dioxide variations at Mauna Loa Observatory](https://onlinelibrary.wiley.com/doi/abs/10.1111/j.2153-3490.1976.tb00701.x)(1976). Článek ukazuje výsledky šestnácti let měření koncentrací CO<sub>2</sub> na Mauna Loa, ukazuje dlouhodobý nárůst koncentrací a je tedy první zprávou o velikosti efektu spalování fosilních paliv na složení atmosféry.   
```
